README.md

```diff
@@ -179,6 +179,16 @@ Or for specfic packages just use `pnpm link @fuel-ts/<pkg-name>`, ex;
 $ pnpm link --global @fuel-ts/wallet
 ```
 
+## Updating Forc version
+
+The following script will upgrade Forc to the latest version on GitHub, remove all lockfiles so the latest stdlib can be used, and rebuild all projects:
+
+```sh
+$ pnpm forc:update
```

Comment:
```suggestion
pnpm forc:update
```

Reply:
Moved to: https://github.com/FuelLabs/fuels-ts/pull/313